caim_base/views/comments.py

```diff
@@ -87,3 +91,51 @@ def delete(request, comment_id):
                 "animal": animal,
             },
         )
+
+
+class CreateSubComment(LoginRequiredMixin, CreateView):
+    model = AnimalSubComment
+    fields = ["body"]
+
+    def form_valid(self, form):
+        if self.request.is_ajax():
```

Comment:
https://stackoverflow.com/questions/63629935/django-3-1-and-is-ajax is_ajax is depreciated > django 3.1